source/user-guide/certificate-import-windows.html.md.erb

```diff
@@ -0,0 +1,109 @@
+---
+title: How to import SSL certificates into AWS Certificate Manager on Windows
+last_reviewed_on: 2022-03-24
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+
+If you are not using a standard Modernisation Platform domain for your app, you will need to import your SSL certificate into your AWS account and apply it to your infrastructure. This guide **does not** cover the infrastructure element of this, only the import of SSL certificates into your AWS account.
+
+This assumes you have a basic understanding of SSL concepts, and access to openssl on the command line.
+
+The steps are:
+
+* Generate a certificate signing request (CSR)
+* Request a certificate using that CSR
+* Complete the certificate request
+* Export the complete certificate
+* Import the exported certificate data to your AWS account
+* Apply the certificate ARN to your app
+
+If you already have your exported SSL certificate in PFX format, skip direct to step 5 (certificate data extraction).
+
+This guide was written using Windows to generate a certificate signing request (CSR) and complete the request. For Mac / Linux, the process will be similar, but you will need to research the commands for creating and completing a CSR, and ideally raise a PR to update this documentation to help other users. 
+
+**There may be a better way**. If so, let us know!
+
+## 1. Generate the CSR (Certificate Signing Request)
+
+Use PowerShell to execute the `certreq -new` command.
+
+(The `performance-hub` repo has a PowerShell script which wraps the `certreq -new` command to generate a CSR and save it to a text file: `.\lib\generateCSR.ps1`)
+
+## 2. Request certificate
+
+Email certificates@digital.justice.gov.uk with the CSR(s) and domain(s), and they will send the certificates.
+
+Note these will be just text files with the certificate data - no private key, and no certification chain (this is important later).
```

Comment:
What do you mean by "text files with the certificate data" - is this just the public key for the cert being requested?

Reply:
This is a PEM-formatted identity certificate that has been digitally signed using the private key of the certificate authority.

To clarify on the above, I have now replaced the comment with the following note:

Note these will be plain text files with the body of the PEM-formatted server certificate from your certificate authority - no private key, and no certification chain (this is important later). The body of the PEM-formatted server certificate is an identity certificate that has been digitally signed using the private key of the certificate authority.

I hope this makes it somehow clearer.